.gitignore

```diff
@@ -0,0 +1,3 @@
+ve/
+.cache/
+seaworthy/__pycache__/
```

Comment:
Probably just want `__pycache__/` (git searches recursively, and Python (3) can put those directories all over the place)